1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,18 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+    if (n === 0) {
```

Comment:
Nice implementation! :)
Just keep an eye on indentation, to make the code more readable.